community/examples/gpu-vm.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: accelerator-demo
+
+# Please review https://cloud.google.com/compute/docs/regions-zones
+# for availability of A2 machine types
+vars:
+  project_id:  ## Set project id here
+  deployment_name: simple
+  region: us-west4
+  zone: us-west4-b
+
+deployment_groups:
+- group: primary
+  modules:
+  - source: modules/network/pre-existing-vpc
+    id: network1
+    kind: terraform
+  - source: modules/scripts/startup-script
+    id: startup-example
+    kind: terraform
+    settings:
+      runners:
+      - type: shell
+        destination: install-qsim.sh
+        content: |
+          #!/bin/bash
```

Comment:
This has me wondering if we should just provide a script for qsim_setup.sh that can be maintained independently of the blueprint. It might be easier to segment maintenance responsibilities and would make the blueprint cleaner.

Reply:
this would only work if you added this to to the startup_script module.

Reply:
if we were to take this approach, what I would do is add a script to the qsim repo and `curl` it as part of the startup script. Similar to the GPU driver installation.

For the moment, though, I prefer the Dockerfile/self-contained approach here in the blueprint. Nothing about qsim requires or prefers conda. This is just a useful (and OS-portable) way of installing nvcc, CUDA and other libraries.

Reply:
Things I feel:
- I appreciate the transparency of the installation.
- I have a lingering concern about ownership of it installation. 
- I like your proposal @tpdownes as a long term solution.